_episodes/02-importing-data.md

```diff
@@ -31,11 +31,12 @@ keypoints:
 >1. Once OpenRefine is launched in your browser, click `Create Project` from the left hand menu and select `Get data from This Computer`
 >2. Click `Choose Files` (or 'Browse', depending on your setup) and locate the file which you have downloaded called `doaj-article-sample.csv`
 >3. Click `Next >>` - the next screen (see below) gives you options to ensure the data is imported into OpenRefine correctly. The options vary depending on the type of data you are importing.
->4. Click in the `Character encoding` box and set it to `UTF-8`
+>4. Click in the `Character encoding` box and set it to `UTF-8` as it will ensure the special characters in the Author column are displaying correctly
 >5. Ensure the first row is used to create the column headings by checking the box `Parse next 1 line(s) as column headers`
->6. Make sure the `Parse cell text into numbers, dates, ...` box is not checked, so OpenRefine doesn't try to automatically detect numbers
->7. The Project Name box in the upper right corner will default to the title of your imported file. Click in the `Project Name` box to give your project a different name, if desired.
->8. Once you are happy click the `Create Project >>` button at the top right of the screen. This will create the project and open it for you. Projects are saved as you work on them, there is no need to save copies as you go along.
+>6. OpenRefine will automatically select "Use character" to enclose cells containing column separators as this will place data in one cell where the values are enclosed in quotes from the source dataset
+>7. Make sure the `Parse cell text into numbers, dates, ...` box is not checked, so OpenRefine doesn't try to automatically detect numbers as it may cause errors such as confusion between American and British date formats
```

Comment:
I think you could just say "confusion between date formats" (as the date format confusion isn't limited to US vs UK!), and I'd add in another example for numbers - perhaps "interpreting ISBNs as numbers, resulting in leading zeros being lost"